### PR TITLE
Check if changelog is present

### PR DESCRIPTION
**Pull request by K-Kumar-01**
## Description

Added the code to check if the repository contains a changelog file

## Motivation and Context

The issue regarding this request is [here](https://github.com/DXHeroes/dx-scanner/issues/397)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly.
- [X] I have added new practice to practice list in README.md.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts
Comment (adelkahomolova) [inline src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts:13]: Article about changelog doesn't exist yet in our Knowledge Base. Please find a relevant source.

File: src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts
Comment (adelkahomolova) [inline src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts:11]:
```suggestion
    'Add a Changelog file to tell other people what changed in the last release.',
```

**Review by adelkahomolova — CHANGES_REQUESTED** [review]

**Comment by adelkahomolova** [comment]
Hi, @K-Kumar-01 , thank you for your pull request. There are just a few small issues that need to be fixed. :)

File: src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts
Comment (K-Kumar-01) [reply src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts:13]: Ok will do this change and resubmit

**Comment by K-Kumar-01** [comment]
> Hi, @K-Kumar-01 , thank you for your pull request. There are just a few small issues that need to be fixed. :)

Apply from the suggestions mentioned, what else needs to be fixed?

File: src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts
Comment (adelkahomolova) [reply src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts:13]: Just this one should be fixed :)

**Comment by adelkahomolova** [comment]
@K-Kumar-01 And also solve the merge conflict, please :)

File: src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts
Comment (K-Kumar-01) [reply src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts:13]: https://www.freecodecamp.org/news/a-beginners-guide-to-git-what-is-a-changelog-and-how-to-generate-it/

will this work ?

Comment (adelkahomolova) [reply src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts:13]: Yes, this article seems fine to me :)

Comment (K-Kumar-01) [reply src/practices/LanguageIndependent/ChangelogIsPresentPractice.ts]: I have updated the link for the article

**Comment by K-Kumar-01** [comment]
> @K-Kumar-01 And also solve the merge conflict, please :)

I have tried resolving the merge conflicts.
Do let me know if i made some mistake.

**Review by adelkahomolova — APPROVED** [review]

**Comment by adelkahomolova** [comment]
Hi, @K-Kumar-01 , I appreciate the time you invest in our repository. I hope you enjoyed the contributing to DX-Scanner! It's approved by me. I'll just wait for a review from my colleague as we think it's the best practice to have 2 approvals to keep the high quality of code.
If you'll want to continue in contributing in open-source, check our other issues on [dx-scanner](https://github.com/DXHeroes/dx-scanner/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc). If you're good at writing, (or you just want to self-education) check our open-source [Knowledge Base](https://developerexperience.io/) (here are the [issues](https://github.com/DXHeroes/knowledge-base-content/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) )
Take care of yourself in these complicated days! ❤️

**Review by Jakub-Vacek — APPROVED** [review]

**Pull request merged**
